datus/cli/screen/workflow_screen.py

```
@@ -506,11 +503,7 @@ def action_select_node(self):
                 # Update the node details panel
                 self.update_node_details()
         except Exception as e:
-            import traceback
-
-            with open("workflow_error.log", "a") as f:
-                f.write(f"Error in action_select_node: {str(e)}\n")
-                f.write(traceback.format_exc())
+            logger.error(f"Error in action_select_node: {e}")
```

Ruff (0.14.4)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)